app/models/procedure.rb

```diff
@@ -128,11 +132,88 @@ def publish_or_reopen!(administrateur)
     end
   end
 
+  def csv_export_stale?
+    !csv_export_file.attached? || csv_export_file.created_at < 3.hours.ago
+  end
+
+  def xlsx_export_stale?
+    !xlsx_export_file.attached? || xlsx_export_file.created_at < 3.hours.ago
+  end
+
+  def ods_export_stale?
+    !ods_export_file.attached? || ods_export_file.created_at < 3.hours.ago
```

Comment:
Pinaillage : je proposerais bien une méthode (ou une constante) `export_expiration_delay` : ça permettrait de factoriser légèrement le code, tout en rendant clair que les exports expirent.

```suggestion
    !ods_export_file.attached? || ods_export_file.created_at < export_expiration_delay.ago
```

Reply:
:+1: j'ai ajouté une constante